src/directives/ctr-input.ts

```diff
@@ -149,7 +149,7 @@ export class CtrInput {
     }
 
     @HostListener("focus", ["$event"])
-    public onfocus() {
+    public onfocus($event) {
```

Comment:
You pass the parameter in ["$event"] to the function but the function doesn't accept it:

```
Error: Error at [...]/ng2-completer/directives/ctr-input.ngfactory.ts:93:35: Supplied parameters do not match any signature of call target.
```

The compiled result in ctr-input.ngfactory.ts:93:35 looks like this:

```
const pd_sub_3:any = ((<any>this.context.onfocus($event)) !== false);
```

I.e. it passes a parameter to a function that doesn't accept one.